src/oaklib/implementations/semsimian/semsimian_implementation.py

```diff
@@ -22,6 +20,8 @@
 from oaklib.interfaces.semsim_interface import SemanticSimilarityInterface
 from oaklib.types import CURIE, PRED_CURIE
 
+Semsimian: Optional["Semsimian"] = None  # type: ignore
```

Comment:
sorry... still pretty confusing. So this is a global variable? With a leading uppercase that makes it sound like a class?

Reply:
It is a class but it is instantiated as a `None`. The class only exists once it is imported from the `semsimian` package. I know it is super confusing. I still prefer my previous solution but this one works too. Mimicking `Gilda` comes with complications like this because we are caching the Semsimian object but Gilda does not cache anything. So something that works for Gilda implementation doesn't necessaarily seem to work for Semsimian implementation.